build/crds/build-crds.sh

```diff
@@ -48,7 +48,7 @@ copy_ob_obc_crds() {
 
 generating_crds_v1() {
   echo "Generating ceph crds"
-  "$CONTROLLER_GEN_BIN_PATH" "$CRD_OPTIONS" paths="./pkg/apis/ceph.rook.io/v1" output:crd:artifacts:config="$OLM_CATALOG_DIR"
+  "$CONTROLLER_GEN_BIN_PATH" "$CRD_OPTIONS" paths="./pkg/apis/ceph.rook.io/v1" "output:crd:artifacts:config=$OLM_CATALOG_DIR"
```

Comment:
Does this change need to be part of this PR?

Reply:
No, there is an intermittent bug here though, I've posted in another comment that if the `mktemp` command produces a file with something that could be parsed as octal, then the crd tool you use doesnt work.

I will revert this change, and perhaps raise a related issue for this. Although I am not sure how it could be addressed.